### ISSUE TITLE
Console errors and no animation when we turn the token

### ISSUE THREAD

**Issue by Grards** [itamarcu/AlternativeRotation#7]
When we use Shift + Ctrl in v.0.8.6 of Foundry, to turn the tokens, we receive this error in the console.
The token turn at the end but we have no animation during the rotation.

![Capture](https://user-images.githubusercontent.com/26524637/120799917-a06c5d80-c53f-11eb-8023-5b2c5407d51e.JPG)

**Comment by itamarcu** [comment]
@Grards - Thanks for the detailed report (the console screenshot was very helpful).  I wasn't able to reproduce this bug but I attempted to add a fix on v1.2.2 that should prevent it from happening.  Please try updating the module and telling me if it works!

(technical details:  the `visualEffectsGraphics` is seemingly null, even though it should have been defined in the `canvasInit` hook. 
 my fix is to just redefine it if needed, whenever it's used)

**Comment by Grards** [comment]
Hello !

I've just tested the v.1.2.2 of AlternativeRotation. And I have no more error in the console. The token turns as he must.
Great job ! And thank you for this highly appreciate module !

**Comment by Grards** [comment]
Hello again ! 

Well, I have another issue in the console when I turn the tokens :

![image](https://user-images.githubusercontent.com/26524637/122187863-bb1fca00-ce8f-11eb-9c56-bb57143f2ea4.png)

But the token turns well. It's just an error in the console this time.

Tested on Google Chrome, version 91.0.4472.106 (Build officiel) (64 bits)
Foundry 0.8.6
Sandbox System 0.9.7

**Comment by itamarcu** [comment]
1. Are you using the "Fast Preview" setting?
2. Does this always happen, or only sometimes (e.g. only after switching scenes, only after selecting a tile)
